samples/CMakeLists.txt

```diff
@@ -20,4 +20,9 @@ add_subdirectory(rtc_timer)
 add_subdirectory(thermistor)
 add_subdirectory(timer)
 add_subdirectory(spi)
-add_subdirectory(lcd)
+add_subdirectory(lcd/basic)
```

Comment:
All of these samples were important for experimenting with the LCD, but now that we have the LCD working the way we want, do you think we need all of them? Are there any you could consolidate?

Reply:
We could totally consolidate a few of them. Specifically Alphabet and ball which are combined into one in the basic example. We could also combine counter and segment as we can just have one of the segments count in that sample.

Should I go through and consolidate these samples?